python_modules/libraries/dagster-databricks/dagster_databricks/databricks_pyspark_step_launcher.py

```diff
@@ -325,6 +326,18 @@ def log_compute_logs(self, log: DagsterLogManager, run_id: str, step_key: str) -
                 f" {step_key}. Check the databricks console for more info."
             )
 
+    def log_run_page_url(self, context: StepExecutionContext, databricks_run_id: int):
+        run = self.databricks_runner.client.workspace_client.jobs.get_run(databricks_run_id).as_dict()
+        run_page_url = run.get("run_page_url")
+        yield DagsterEvent.from_step(
```

Comment:
I would add a `message` here. This message could just be `"Waiting for Databricks run %s to complete..."`, and replace the regular log call that does that.

Reply:
Great idea! Updated accordingly 👍